### PR TITLE
Revert "(packaging) Update shipping defaults to work with new puppet5…

### PR DESCRIPTION
**Pull request by melissa**
… repos"

This reverts commit 26d807b15ac7cff1a84f226189b19253e26bc3fd.

### CODE REVIEW

File: configs/components/puppet.json [hunk 1]
```diff
@@ -1 +1 @@
-{"url": "git://github.com/puppetlabs/puppet.git", "ref": "7db03e5a28e9c2499da6e34d6c38849cce399b12"}
+{"url": "git://github.com/puppetlabs/puppet.git", "ref": "d21c5a749a87cf4e4308880e5daf331ccf8fd49e"}
```
Comment (geoffnichols) [inline configs/components/puppet.json]: Ah, it looks like retargeting this updated the puppet SHA.

Comment (McdonaldSeanp) [reply configs/components/puppet.json]: Yeah, we will need to keep that update out of this PR

Comment (caseywilliams) [reply configs/components/puppet.json]: @melissa Would you mind rebasing this against the `5.3.1_release` branch?

Comment (melissa) [reply configs/components/puppet.json]: done.

**Pull request merged**